x-pack/plugins/apm/public/components/shared/charts/timeseries_chart.tsx

```diff
@@ -118,19 +110,45 @@ export function TimeseriesChart({
       ? anomalyChartTimeseries?.boundaries
       : []),
     ...(anomalyChartTimeseries?.scores ?? []),
-  ];
+  ]
+    // Sorting series so that area type series are before line series
+    // This is a workaround so that the legendSort works correctly
+    .sort(
```

Comment:
This is a workaround this issue https://github.com/elastic/elastic-charts/issues/1685